java/ql/src/experimental/Security/CWE/CWE-548/InsecureDirectoryConfig.ql

```diff
@@ -0,0 +1,42 @@
+/**
+ * @id java/j2ee-server-directory-listing
+ * @name Inappropriately exposed directories and files yielding sensitive information like source code and credentials to attackers.
```

Comment:
This name is a bit long.  Also could you move the `@id` a couple of lines down - we usually start with `@name` and `@description` as the first two lines (cf. other queries).

Reply:
Thanks @aschackmull for reviewing this PR. I've shortened the `@id` attribute value and moved the attribute below `@name` and `@description`.

Reply:
The length of the `@id` was fine - now it's a bit short and too vague. It was the `@name` that needed shortening.

Reply:
Sorry if I wasn't being clear in my original comment.

Reply:
Thanks @aschackmull . I've corrected the `@name` and `@id` attributes, as well as updated the `@description` attribute.